src/relstorage/_util.py

```diff
@@ -604,3 +608,12 @@ def __repr__(self):
             self.needs_fetchall,
             self.last_stmt,
         )
+
+
+
+try:
+    hashed = md5_original('test'.encode())
+    md5 = md5_original
+except ValueError:
```

Comment:
```suggestion
except ValueError: # pragma: no cover
```

We don't expect to ever test this branch.